form/form_test.go

```diff
@@ -147,9 +169,28 @@ func (s *formSuite) TestIOPrompter(c *gc.C) {
 	}
 }
 
+func (s *formSuite) TestIOPrompterWrtiteErrors(c *gc.C) {
+	prompter := form.IOPrompter{
+		In:  nil,
+		Out: errorWriter{},
+	}
+	err := prompter.ShowTitle("title")
+	c.Assert(err, gc.ErrorMatches, "cannot show title: test")
+	_, err = prompter.Prompt("attr", environschema.Attr{
+		Description: "A",
+	})
+	c.Assert(err, gc.ErrorMatches, "cannot write prompt: test")
```

Comment:
I'd like to see a test that PromptingFiller returns an error when ShowTitle does.